documentation/library_guide/conf.py

```diff
@@ -15,7 +15,8 @@
 #===============================================================================
 
 ##  Content:
-##     Intel(R) DAAL configuration file for the Sphinx documentation builder
+## Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL) configuration
```

Comment:
Seems this comment is not actual at all, why should `Intel(R) DAAL` be mentioned in this file?

Reply:
Thanks for noticing! I'll clarify.

Reply:
Looks like Intel(R) DAAL is mentioned here on purpose. Some lines in the script are related to DAAL. Let's keep it.

Reply:
Please check it separately more carefully.